lang/semgrep-grammars/src/semgrep-ruby/grammar.js

```diff
@@ -12,6 +12,14 @@ module.exports = grammar(standard_grammar, {
   rules: {
     /* Ruby global variables start with a '$' ('global_variable'). */
 
+    semgrep_dots: $ => '...',
```

Comment:
they're called semgrep_ellipsis in all of our other grammars, maybe worth keeping it consistent.